go/libcmdline/cmdline.go

```diff
@@ -352,6 +367,18 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Name:  "api-timeout",
 			Usage: "set the HTTP timeout for API calls to the keybase API server",
 		},
+		cli.StringFlag{
+			Name:  "tor-mode",
+			Usage: "set TOR mode to be 'leaky', 'none', or 'strict'. 'none' by default",
```

Comment:
Is there space to document that "strict tries to be anonymous even to the Keybase server, leaky uses Tor for your connection but identifies you to the server"?